mobile/src/navigations/onboarding-navigation/components/step/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from 'react-native';
+
+import { Color } from '~/bundles/common/enums/enums';
+
+const styles = StyleSheet.create({
+    singleStep: {
+        gap: 10,
+    },
+    textCon: {
+        position: 'relative',
+        top: 13,
+    },
+    step: {
+        color: Color.PRIMARY,
+        textTransform: 'uppercase',
+        fontWeight: 'bold',
+        letterSpacing: 2,
+    },
```

Comment:
I made a similar font in globalStyles:
```
[TextCategory.STEP]: {
        FontFamily: FontFamily.INTER_BOLD,
        fontSize: 13,
        color: Color.PRIMARY,
        textTransform: 'uppercase' as const,
    },
```
Let's see if we can use it together.

Reply:
I merged your branch in mine and yes this styles is the same